.evergreen/run-tests.sh

```diff
@@ -120,7 +120,12 @@ if [ "$SLOW_TESTS_ONLY" == "true" ]; then
     ./gradlew -PjdkHome=/opt/java/${JDK} -Dorg.mongodb.test.uri=${MONGODB_URI} \
               ${TRANSACTION_URI} ${GRADLE_EXTRA_VARS} ${ASYNC_TYPE} --stacktrace --info testSlowOnly
 else
+    set +x
+    echo ./gradlew -PjdkHome=/opt/java/${JDK} -Dorg.mongodb.test.uri=${MONGODB_URI} \
```

Comment:
Echoing the command seems to defeat the purpose of this change, no?

Reply:
Ah sorry for the confusion:  the command is merely echoing the literal strings of `AWS_SECRET_ACCESS_KEY`  and `AWS_ACCESS_KEY_ID` (as opposed to echoing the contents of the variable via   `${AWS_ACCESS_KEY_ID}`  and `${AWS_SECRET_ACCESS_KEY}`